internal/trace/trace.go

```diff
@@ -18,27 +18,29 @@ package trace
 import (
 	"context"
 
-	"go.opencensus.io/trace"
+	"go.opentelemetry.io/otel/codes"
```

Comment:
This package doesn't actually appear to be used anywhere; I deleted it so you can drop these files and merge.

Reply:
Rebased